Mobile/ShopLens/Droid/Listeners/SurfaceTextureListener.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Android.Views;
+
+namespace ShopLens.Droid.Listeners
+{
+    public class SurfaceTextureListener : Java.Lang.Object, TextureView.ISurfaceTextureListener
+    {
+        private readonly Camera2Fragment owner;
+
+        public SurfaceTextureListener(Camera2Fragment owner)
+        {
+            if (owner == null)
+                throw new System.ArgumentNullException("owner");
+            this.owner = owner;
+        }
+
+        public void OnSurfaceTextureAvailable(Android.Graphics.SurfaceTexture surface, int width, int height)
+        {
+            owner.OpenCamera(width, height);
+        }
+
+        public bool OnSurfaceTextureDestroyed(Android.Graphics.SurfaceTexture surface)
+        {
+            return true;
+        }
+
+        public void OnSurfaceTextureSizeChanged(Android.Graphics.SurfaceTexture surface, int width, int height)
+        {
+        
```

Comment:
Seems really empty.

Reply:
If the interface requires an implementation but we want to ignore it, leaving the method empty seems reasonable.

Another question is whether we want to ignore it.